### PR TITLE
Update Exim.org.xml

### PR DESCRIPTION
**Pull request by J0WI**
#15750

### CODE REVIEW

File: src/chrome/content/rules/Exim.org.xml [hunk 1]
```diff
@@ -1,10 +1,21 @@
 <!--
-	(www.)?exim.org: Shows bugs.exim.org
+	Nonfunctional hosts in *.exim.org:
+		- wiki.exim.org (m)
```
Comment (Bisaloo) [inline src/chrome/content/rules/Exim.org.xml]: This now works

File: src/chrome/content/rules/Exim.org.xml [hunk 2]
```diff
@@ -1,10 +1,21 @@
...
 -->
-<ruleset name="Exim.org (partial)">
+<ruleset name="Exim.org">
 
+	<target host="exim.org" />
+	<target host="www.exim.org" />
 	<target host="bugs.exim.org" />
+	<target host="ftp.exim.org" />
+	<target host="git.exim.org" />
 	<target host="lists.exim.org" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/Exim.org.xml]: vcs.exim.org

File: src/chrome/content/rules/Exim.org.xml [hunk 3]
```diff
@@ -1,10 +1,21 @@
...
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
 -->
-<ruleset name="Exim.org (partial)">
+<ruleset name="Exim.org">
 
+	<target host="exim.org" />
+	<target host="www.exim.org" />
 	<target host="bugs.exim.org" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/Exim.org.xml]: buildfarm.exim.org
downloads.exim.org

File: src/chrome/content/rules/Exim.org.xml (hunk 1 above)
Comment (Bisaloo) [inline src/chrome/content/rules/Exim.org.xml]: docs.exim.org (m)

File: src/chrome/content/rules/Exim.org.xml (hunk 3 above)
Comment (J0WI) [reply src/chrome/content/rules/Exim.org.xml]: Added, thanks!

**Comment by Bisaloo** [comment]
thank you!

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**